Sentinel/Classes/Core/Internal/SentinelTabBarController.swift

```diff
@@ -28,3 +36,15 @@ extension SentinelTabBarController {
         self.viewControllers = viewControllers
     }
 }
+
+// MARK: - Private methods -
+
+private extension SentinelTabBarController {
+
+    func setPreselectedActionIfNeeded() {
+        defer { didPreselectAction = true }
+        guard !didPreselectAction else { return }
+        // Preselect Tools tab
+        selectedIndex = 2
```

Comment:
Would rather use enum for tab screens with index as raw value if possible, you wouldn't need comment above then :)

Reply:
Let's leave it for now, but I'll open todo on Productive for this to be handled in the future :)